fastlane/lib/fastlane/actions/check_app_store_metadata.rb

```diff
@@ -7,7 +7,6 @@ class CheckAppStoreMetadataAction < Action
       def self.run(config)
         require 'precheck'
         Precheck.config = config
-        Precheck.config[:api_key] ||= Actions.lane_context[SharedValues::APP_STORE_CONNECT_API_KEY]
```

Comment:
Not needed anymore because `Precheck` will look for existing `Spaceship::ConnectAPI.client` now and use that